rebar.lock

```diff
@@ -1,4 +1,4 @@
-{"1.1.0",
+{"1.2.0",
```

Comment:
Bumped when getting deps with rebar3.

Reply:
I have exactly the same bump at: https://github.com/aws-beam/aws_credentials/pull/19